feature/google/src/bc5SupportDev/java/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -195,6 +197,71 @@ class BillingWrapper(
         }
     }
 
+    override fun queryProductDetailsAsync(
```

Comment:
did you mean to leave the `type` out as a parameter here?

Reply:
Yes, i had it hardcoded for some reason, we'll keep it